wire/blockheader_test.go

```diff
@@ -216,14 +217,15 @@ func TestBlockHeaderWire(t *testing.T) {
 			continue
 		}
 
-		b, err := wire.TstBytesBlockHeader(test.in)
+		buf.Reset()
+		err = test.in.BtcEncode(&buf, pver)
```

Comment:
We should probably drop BtcEncode at some point for something more generic... (ccEncode??) But not a consideration for current PR.